### PR TITLE
re-adding Ubuntu install

### PR DESCRIPTION
**Pull request by aaronhmiller**
**NOTE**: Please read the CONTRIBUTING.md guidelines before submitting your patch,
and ensure you followed them all:

https://github.com/Kong/getkong.org/blob/master/CONTRIBUTING.md

### Summary

Saw that Ubuntu was missing from right-hand install menu

### Full changelog

* ported 0.31.x info into 0.32

### Issues resolved
missing install doc

### Checklist:

- [/] [Commit message & atomicity](https://github.com/Kong/getkong.org/blob/master/CONTRIBUTING.md#commit-atomicity) checked
- [/] Documentation 
- [/] Spellchecked my updates. Plus I rarely mis-spell
- [/] Ready to be merged

### CODE REVIEW

**Comment by aaronhmiller** [comment]
thanks for the thoughtful input

**Comment by darrenjennings** [comment]
Does 0.31 need ubuntu instuctions as well?

File: app/docs/enterprise/0.32-x/installation/ubuntu.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
+---
+title: How to install Kong Enterprise and PostgreSQL onto Ubuntu
+---
+
+## Install Kong Enterprise
+
+curl the EE binary for Ubuntu (or Debian) from Bintray to your Ubuntu server
```
Comment (darrenjennings) [inline app/docs/enterprise/0.32-x/installation/ubuntu.md]: I think we should have a code block after this with an example `curl` statement.

File: app/docs/enterprise/0.32-x/installation/ubuntu.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+# Add contents of your license file (copy & paste)
+$ sudo vi /etc/kong/license.json
+$ sudo cp /etc/kong/kong.conf.default /etc/kong/kong.conf
+# Uncomment and add 'kong' to pg_password line
+$ sudo vi /etc/kong/kong.conf
+```
+
+### Run migrations and start kong
+```bash
+$ sudo /usr/local/bin/kong migrations up
```
Comment (darrenjennings) [inline app/docs/enterprise/0.32-x/installation/ubuntu.md]: probably should add this for 0.31 as well?

Comment (aaronhmiller) [reply app/docs/enterprise/0.32-x/installation/ubuntu.md]: hmm, you mean like "Ubuntu" here: https://getkong.org/docs/enterprise/0.31-x/ which links to here: https://getkong.org/docs/enterprise/0.31-x/installation/ubuntu/

So, already in 0.31?

Comment (darrenjennings) [reply app/docs/enterprise/0.32-x/installation/ubuntu.md]: It looks like this command changed, so I'm commenting that 0.31 should include this change as well?

`kong migrations up` -> `sudo /usr/local/bin/kong migrations up`

Comment (aaronhmiller) [reply app/docs/enterprise/0.32-x/installation/ubuntu.md]: Thank you eagle eyes!

**Pull request merged**